release/release_build.py

```diff
@@ -326,80 +324,84 @@ def main(argv):
 
         if projects_to_release[AFU_OPT]:
-            propose_changelog_edit( "Annotation File Utilities", AFU_CHANGELOG, TMP_DIR + "/afu.changes",
-                                    old_afu_version, ANNO_TOOLS, AFU_TAG_PREFIXES  )
+            propose_change_review( "the Annotation File Utilities documentation updates", old_afu_version, ANNO_TOOLS,
+                                   AFU_TAG_PREFIXES, AFU_MANUAL, TMP_DIR + "/afu.manual"  )
 
         if projects_to_release[CF_OPT]:
-            propose_changelog_edit( "Checker Framework", CHECKER_CHANGELOG, TMP_DIR + "/checker-framework.changes",
-                                    old_jsr308_version, CHECKER_FRAMEWORK, CHECKER_TAG_PREFIXES )
-
-    # This step will write out all of the changes that happened to the individual projects' manuals
-    # to a temporary file. Please review these changes for errors. You can edit them in the build
-    # repository and they will be committed by the release_push.py script.
+            build_checker_framework_release(False, jsr308_version, afu_date, "", "", manual_only = True)
+
+            print("")
+            print("The built Checker Framework manual (HTML and PDF) can be found at " + CHECKER_MANUAL)
+            print("")
+            print("Verify that the manual PDF has no lines that are longer than the page width")
+            print("(it is acceptable for some lines to extend into the right margin).")
+            print("")
+            print("If any checkers have been added or removed, then verify that the lists")
+            print("of checkers in these manual sections are up to date:")
+            print(" * Introduction")
+            print(" * Run-time tests and type refinement")
+            print("and make sure that the checkers supported in the Eclipse plug-in are up to date")
+            print("by following the instructions at eclipse/README-developers.html#update_checkers")
+            print("")
+
+            propose_change_review( "the Checker Framework documentation updates", old_jsr308_version, CHECKER_FRAMEWORK,
+                                   CHECKER_TAG_PREFIXES, CHECKER_MANUAL, TMP_DIR + "/checker-framework.manual"  )
 
-    print_step("Build Step 5: Review manual changes.") # SEMIAUTO
+        return
 
-    print("If any checkers have been added or removed, then verify that the lists")
-    print("of checkers in these manual sections are up to date:")
-    print(" * Introduction")
-    print(" * Run-time tests and type refinement")
+    print_step("Build Step 4: Copy entire live site to dev site (~22 minutes).") # AUTO
 
-    print("Verify that the manual PDF has no lines that are longer than the page width")
-    print("(it is acceptable for some lines to extend into the right margin).")
+    # TODO: Ask support to delete the dev_old directory as it has files I cannot delete or chmod, then remove "--exclude=dev_old" from the command below.
```

Comment:
Request submitted. I now have enough confidence to delete the old dev site after having tested the build script a few times.